common/models/user.js

```diff
@@ -669,6 +669,42 @@ module.exports = function(User) {
       next();
     });
 
+    // Delete old sessions once email is updated
+    UserModel.observe('before save', function beforeEmailUpdate(ctx, next) {
+      var oldEmail, newEmail, oldId, newId;
+      if (ctx.isNewInstance) return next();
+
+      if (ctx.where) {
+        ctx.hookState.id = ctx.where.id;
+        oldId = ctx.hookState.id;
+      }
```

Comment:
How about the case when `ctx.where` is not set and there is `ctx.instance` instead?